packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -4624,10 +4668,17 @@ class SemanticsConfiguration {
     if (!child.hasBeenAnnotated) {
       return;
     }
-
-    _actions.addAll(child._actions);
+    if (child.isBlockingUserInteractions) {
+      child._actions.forEach((SemanticsAction key, SemanticsActionHandler value) {
+        if (_kSystemInteractions & key.index > 0) {
+          _actions[key] = value;
+        }
+      });
+    } else {
+      _actions.addAll(child._actions);
+    }
```

Comment:
I added this filtering because I was trying to support use case where merging child that has same action but blocked. I guess that is no longer the case.

Even without that requirement, why do we want to add a action to the map that will be blocked anyway?

Reply:
It's fine to do that. I was just thinking if we don't have to do it what's the point of implementing all this extra logic?